packages/office-addin-lint/src/lint.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {logErrorMessage} from "office-addin-cli";
+import * as defaults from "./defaults";
+
+function execCommand(command: string) {
+    const execSync = require('child_process').execSync;
+    const child = execSync(command, { stdio: 'inherit' });
+}
+
+export function lintCheckCommand(files: string): string {
+  const eslintCommand: string = "eslint " + files;
+  return eslintCommand;
+}
+
+export function lintCheck(files: string) {
```

Comment:
rename to `performLintCheck`.  I think the verb just helps it be more descriptive.